src/lxc/cgroups/cgfsng.c

```diff
@@ -1378,7 +1396,24 @@ static inline bool cgfsng_create(void *hdata)
 	return false;
 }
 
-static bool cgfsng_enter(void *hdata, pid_t pid)
+static inline bool cgfsng_create_inner(struct cgfsng_handler_data *d)
+{
+	size_t i;
+	bool ret = true;
+	char *cgname = must_make_path(d->container_cgroup, "ns", NULL);
```

Comment:
Also, maybe you can make this a `#define` somewhere and add a little comment. :)